We need to run the t3404 test without this patch on Linux to see how it behaves
there.  If it pass there then we need to find why.  If it fails there then we
need to submit this (or similar) fix upstream.

--- git-2.45.1/t/t3404-rebase-interactive.sh.orig
+++ git-2.45.1/t/t3404-rebase-interactive.sh
@@ -1916,13 +1916,14 @@
 	test_cmp_rev HEAD~1 refs/heads/third &&
 	test_cmp_rev HEAD refs/heads/no-conflict-branch &&
 
-	cat >expect <<-\EOF &&
+	tab=$'\''\t'\'' &&
+	cat >expect <<-EOF &&
 	Successfully rebased and updated refs/heads/update-refs.
 	Updated the following refs with --update-refs:
-		refs/heads/first
-		refs/heads/no-conflict-branch
-		refs/heads/second
-		refs/heads/third
+	${tab}refs/heads/first
+	${tab}refs/heads/no-conflict-branch
+	${tab}refs/heads/second
+	${tab}refs/heads/third
 	EOF
 
 	# Clear "Rebasing (X/Y)" progress lines and drop leading tabs.
@@ -2177,13 +2178,14 @@
 	test_must_fail git rebase --continue 2>err &&
 	grep "update_ref failed for ref '\''refs/heads/second'\''" err &&
 
-	cat >expect <<-\EOF &&
+	tab=$'\''\t'\'' &&
+	cat >expect <<-EOF &&
 	Updated the following refs with --update-refs:
-		refs/heads/first
-		refs/heads/no-conflict-branch
-		refs/heads/third
+	${tab}refs/heads/first
+	${tab}refs/heads/no-conflict-branch
+	${tab}refs/heads/third
 	Failed to update the following refs with --update-refs:
-		refs/heads/second
+	${tab}refs/heads/second
 	EOF
 
 	# Clear "Rebasing (X/Y)" progress lines and drop leading tabs.
